config.json

```
@@ -353,6 +345,20 @@
         "concepts": ["links", "tasks"],
         "prerequisites": ["processes", "pids", "enum", "errors", "maps"],
         "status": "wip"
+      },
+      {
+        "slug": "lucas-numbers",
+        "name": "Lucas Numbers",
+        "uuid": "b82ecdf0-9739-4e25-a261-aa815f8dbf0d",
+        "concepts": ["streams"],
+        "prerequisites": [
+          "guards",
```

I'm pretty sure we have to require enum for streams, since we're introducing them as "lazy enums" without further explanations :)
```suggestion
          "enum",
          "guards",
```